src/main/java/eu/xenit/gradle/alfrescosdk/tasks/Amp.java

```diff
@@ -52,11 +47,11 @@
 
     private ConfigurableFileCollection deBundles = getProject().files();
 
-    private final DefaultCopySpec ampCopySpec = (DefaultCopySpec) getRootSpec().addChildBeforeSpec(getMainSpec()).into("");;
+    private final DefaultCopySpec ampCopySpec = (DefaultCopySpec) getRootSpec().addChildBeforeSpec(getMainSpec()).into("");
 
     public Amp() {
-        setExtension(AMP_EXTENSION);
-        setDestinationDir(getProject().getBuildDir().toPath().resolve("dist").toFile());
+        getArchiveExtension().set(AMP_EXTENSION);
```

Comment:
@bhagyas This does not work with Gradle 5.0.
@vierbergenlars will be back next week. I'll see with him if it is worth dropping 5.0 support.

Reply:
This works:
        if(GradleVersion.current().compareTo(GradleVersion.version("5.0")) > 0 ){
            getArchiveExtension().set(AMP_EXTENSION);
            getDestinationDirectory().set(getProject().getBuildDir().toPath().resolve("dist").toFile());
        } else {
            setExtension(AMP_EXTENSION);
            setDestinationDir(getProject().getBuildDir().toPath().resolve("dist").toFile());
        }

You should get rid of some of the deprecation warnings for this case, and still support 5.0. However, there are other deprecation warnings. Gradle API's move quickly.